src/resources/postcss/components/_datepicker.pcss

```diff
@@ -0,0 +1,135 @@
+/* -----------------------------------------------------------------------------
+ *
+ * Datepicker
+ *
+ * ----------------------------------------------------------------------------- */
+
+.tribe-events {
+
+	.datepicker {
+		background-color: var(--color-background);
+		border: 1px solid var(--color-border-default);
+		box-shadow: var(--box-shadow-tooltip);
+		margin-top: var(--spacer-1);
+		padding: 0 var(--spacer-3) var(--spacer-3);
+
+		@media (--viewport-medium) {
+			margin-top: var(--spacer-3);
+		}
+
+		&:before,
+		&:after {
+			display: none;
+		}
+
+		td,
+		th {
+			height: initial;
+			width: initial;
+		}
+
+		.dow {
+			color: var(--color-text-primary);
+			font-size: var(--font-size-0);
+			font-weight: var(--font-weight-regular);
+			padding: initial;
+			padding-bottom: var(--spacer-2);
+		}
+
+		.datepicker-switch {
+			color: var(--color-text-primary);
+			font-size: var(--font-size-3);
```

Comment:
same comment as above on line height and using an already-defined mixin.

Reply:
still need line height here.

Reply:
I set it here, with the others: https://github.com/moderntribe/the-events-calendar/pull/2599/files#diff-572931b824026e2e53c9d15dc50c838dR46